src/main/webapp/app/utils/date.utils.ts

```diff
@@ -0,0 +1,18 @@
+import dayjs from 'dayjs/esm';
+
+/**
+ * Converts a date in order to send it to the server
+ * @param date the date either as proper date or string representation
+ */
+export function convertDateFromClient(date?: dayjs.Dayjs): string | undefined {
```

Comment:
Consider adding a `@returns` tag in the JSDoc describing the format of the returned date string